### PR TITLE
New group creation

### PR DESCRIPTION
**Pull request by Shehanka**

### CODE REVIEW

File: src/app/components/events/events.component.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  <div class="container">
+    <div class="row">
+      <div class="col-2"></div>
+      <div class="col-7">
+        <div class="container">
+          <div *ngFor="let event of events">
+            <div class="card mb-3" style="max-width: 540px;">
+              <div class="row g-0">
+                <div class="col-md-4">
+                  <img src="{{event.image}}" alt="..." width="180px"/>
```
Comment (sajeetharan) [inline src/app/components/events/events.component.html:15]: Use safe navigation

File: src/app/components/events/events.component.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                </div>
+                <div class="col-md-8">
+                  <div class="card-body">
+                    <h5 class="card-title">{{event.title}}</h5>
```
Comment (sajeetharan) [inline src/app/components/events/events.component.html:19]: Use safe navigation

File: src/app/components/groups/groups.component.html [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    <button class="btn btn-primary" routerLink="/new-group">
+      Start a new Group
+    </button>
+  </div>
+
+  <div class="groups-container">
+    <div><h5>My Groups</h5></div>
+
+    <!-- If My groups available -->
+    <div *ngIf="isMyGroupsAvailable">
```
Comment (sajeetharan) [inline src/app/components/groups/groups.component.html:16]: Try single line https://stackoverflow.com/questions/43812770/ngif-else-if-in-template

File: src/app/components/new-group/new-group.component.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        type="text"
+        class="form-control"
+        id="name"
+        aria-describedby="name"
+        value="Software Engineering"
+      />
+
+      <div class="container" *ngFor="let topic of topics">
+        <span class="badge rounded-pill bg-primary" style="float: left;">{{
+          topic.name
```
Comment (sajeetharan) [inline src/app/components/new-group/new-group.component.html:30]: use safe navigation operator

File: src/app/components/events/events.component.html (hunk 1 above)
Comment (Shehanka) [reply src/app/components/events/events.component.html:15]: Ohh, i forgot that

**Comment by Shehanka** [comment]
@sajeetharan  All requested Changes Added

File: src/app/components/groups/groups.component.html (hunk 3 above)
Comment (Shehanka) [reply src/app/components/groups/groups.component.html]: Thanks for the tip

**Pull request merged**